internal/test/e2e/e2e_test.go

```diff
@@ -714,4 +715,26 @@ func scaleDownTests(t *testing.T, kubectl *kubectlContext) {
 	)
 	require.NoError(t, err, out)
 	assert.Equal(t, expectedValue+"\n", out)
+
+	t.Log("The cluster can then be scaled back up")
+	err = kubectl.Scale("etcdcluster/my-cluster", expectedReplicasOriginal)
+	require.NoError(t, err)
+	err = try.Eventually(
+		func() error {
+			out, err := kubectl.Get("etcdcluster", "my-cluster", "-o=jsonpath={.status.replicas}")
+			if err != nil {
+				return err
+			}
+			statusReplicas, err := strconv.Atoi(out)
+			if err != nil {
+				return err
+			}
+			if statusReplicas != expectedReplicasOriginal {
+				return fmt.Errorf("unexpected status.replicas. Wanted: %d, Got: %d", expectedReplicasOriginal, statusReplicas)
+			}
+			return nil
+		},
+		time.Minute*2, time.Second*10,
+	)
+	require.NoError(t, err)
```

Comment:
It seems like this test would have passed even prior to this PR? We aren't actually validating that the new member has been successfully connected into the cluster, only that the `status.replicas` field is updated to account for the new replica (which I think will happen anyway even if that peer is crashing)

Reply:
It did fail before this PR, because the cluster gets stuck after adding back the second peer.
But you're right, that if the ETCD API can't be reached (for some reason) then the operator would add peers anyway. 
So I'll change this test to check that data can be written to the cluster once the expected number of replicas is reached.

Reply:
Done.